.github/workflows/continous-integration.yml

```diff
@@ -88,89 +88,119 @@ jobs:
         python-version: [3.6, 3.7]
 
     steps:
-    - name: Checkout git repository 🕝
-      uses: actions/checkout@v2
-
-    - name: Set up Python ${{ matrix.python-version }} 🐍
-      uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-
-    - name: Read Poetry Version 🔢
-      run: |
-           echo "POETRY_VERSION=$(scripts/poetry-version.sh)" >> $GITHUB_ENV
-      shell: bash
-
-    - name: Install poetry 🦄
-      uses: Gr1N/setup-poetry@v4
-      with:
-        poetry-version: ${{ env.POETRY_VERSION }}
-
-    - name: Load Poetry Cached Libraries ⬇
-      uses: actions/cache@v1
-      with:
-        path: ~/.cache/pypoetry/virtualenvs
-        key: ${{ runner.os }}-poetry-${{ matrix.python-version }}-${{ hashFiles('**/poetry.lock') }}
-        restore-keys: ${{ runner.os }}-poetry-
-
-    - name: Install Dependencies 📦
-      run: make install
-
-    - name: Test Code 🔍
-      run: make test
-
-    - name: Send Coverage Report 📊
-      if: matrix.python-version == 3.6
-      env:
-        COVERALLS_REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        COVERALLS_SERVICE_NAME: github
-      run: poetry run coveralls
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }} 🐍
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Read Poetry Version 🔢
+        run: |
+          echo "POETRY_VERSION=$(scripts/poetry-version.sh)" >> $GITHUB_ENV
+        shell: bash
+
+      - name: Install poetry 🦄
```

Comment:
What happened here? Did your IDE change formatting?

Reply:
Didn't realize my IDE formatted it😱

Reply:
Reverted to previous commit.